client/components/tech-update-form.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Form, Button } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+export default function TechUpdateForm({ project, onSave, onCancel }) {
+  const poNumber = project.poNumber;
+  const name = project.name;
+  const address = project.address;
+  const city = project.city;
+  const state = project.state;
+  const zipcode = project.zipcode;
+  const [notes, setNotes] = useState(project.notes);
+  const [completed, setCompleted] = useState(project.completed);
+  const assignedTo = project.assignedTo;
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const req = {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ poNumber, name, address, city, state, zipcode, notes, completed, assignedTo })
```

Comment:
You could do this a lot easier with:
`const newProject = { ...project, notes, completed };`

`...project` says "Copy all the fields from `project`  into `newProject`", then you overwrite `project.notes` with `notes` and then `project.completed` with `completed`.

Then you wouldn't need to extract all the fields up above. Even nicer, if you add fields to `project` in the future, they'll all be copied over without any code modifications.